x/ecocredit/location_test.go

```diff
@@ -0,0 +1,79 @@
+package ecocredit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidateLocation(t *testing.T) {
+	specs := []struct {
+		name      string
+		location  string
+		expectErr bool
+	}{
+		{
+			name:      "can validate valid country code",
```

Comment:
```suggestion
			name:      "valid country code",
```
we can drop the "can/can't validate" for better readability